governance/program/src/instruction.rs

```diff
@@ -204,6 +204,13 @@ pub enum GovernanceInstruction {
         #[allow(dead_code)]
         /// Unique seed for the Proposal PDA
         proposal_seed: Pubkey,
+
+        #[allow(dead_code)]
+        /// Indicates whether space for the proposal account will be taken from data provided through the call
+        /// (when None) or the space for the account will be pre-defined by some amount
+        /// Prefetching the space can be used for adding options into the proposal in separate instruction when in draft state
+        /// Value 0 means no prefetching to happen.
+        prefetch_space: u64,
```

Comment:
Since `solana` runtime now supports resizing of the accounts I think it'll be better to resize the account as we add new options instead of allocating it upfront